packages/css/src/components/buttons.css

```diff
@@ -0,0 +1,138 @@
+@layer components {
+  /* Default button */
+  .vtmn-btn {
+    --button-font-family: var(--vtmn-typo_font-family--condensed), Arial,
+      Helvetica, sans-serif;
+    --button-font-size: 16px;
```

Comment:
Do you consider working with `em` or `rem` in the future?

Reply:
We are working with that because designers elements are defined like that. 
You think it could be interesting to declare it for the whole application ?

Reply:
I guess it should be defined as some global level and defined in em unit. That's a very good practice that we should not avoid even if designers work in px (and that also a lot of design system work like this too...).
As a concrete example, while zooming on the e-commerce website, that would imply that everything adapts except buttons coming from design system 😕

Check out this interesting article https://medium.com/codyhouse/create-your-design-system-part-1-typography-7c630d9092bd

Reply:
Thanks @thollander for this suggestion.
Are you sure that zoom is always a problem for modern browsers?
But I understand why it's important to use relative units like rem,em instead of absolute like px,cm etc.
I will this from Tailwind: https://tailwindcss.com/docs/font-size#font-sizes

Reply:
I've just made an issue for this refactoring: #19

Reply:
Hey,

We should use rem or em, as Terence said.

It's a best practice to use the value defined in the browser (by default, 16px). 

W3C recommandation:
 https://www.w3.org/TR/WCAG20-TECHS/C14.html